packages/config/src/tokens/tokenList.json

```diff
@@ -2912,6 +2912,20 @@
       "type": "CBV",
       "formula": "locked"
     },
+    {
+      "id": "wagmi-wagmi",
+      "name": "Wagmi",
+      "coingeckoId": "wagmi-2",
+      "address": "0x92CC36D66e9d739D50673d1f27929a371FB83a67",
+      "symbol": "WAGMI",
+      "decimals": 18,
+      "sinceTimestamp": 1688483015,
```

Comment:
sinceTimestamp for this token should be changed to 1695376800 because this is the time coingecko started to tracking its price

Reply:
Ok, but how do you know that?